lms/djangoapps/grades/tasks.py

```diff
@@ -47,6 +48,21 @@
 RECALCULATE_GRADE_DELAY = 2  # in seconds, to prevent excessive _has_db_updated failures. See TNL-6424.
 
 
+@task(max_retries=2, default_retry_delay=10, backend=ChordableDjangoBackend(current_app))
+def add(x, y):
+    if x is 2:
+        raise add.retry(x=x, y=y, exc=NotImplementedError("WASTED"))
+    return x + y
+
+
+@task(backend=ChordableDjangoBackend(current_app))
```

Comment:
@yro I had been forgetting that the callback task can specify a backend in the `task` decorator, that solved all our problems